set.mm

```diff
@@ -656377,6 +656384,24 @@ D Fn ( I ... ( M - 1 ) ) /\
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 $)
 
+  ${
+    ressbasssg.r $e |- R = ( W |`s A ) $.
+    ressbasssg.b $e |- B = ( Base ` W ) $.
+    $( The base set of a restriction to ` A ` is a subset of ` A ` and the base
+       set ` B ` of the original structure.  (Contributed by SN,
+       10-Jan-2025.) $)
+    ressbasssg $p |- ( Base ` R ) C_ ( A i^i B ) $=
+      ( cvv wcel cbs cfv cin wss ressbas ssid eqsstrrdi wn c0 cress reldmress
+      co ovprc2 eqtrid fveq2d base0 0ss eqsstrri eqsstrdi pm2.61i ) AGHZCIJZABK
+      ZLUIUJUKUKABCGDEFMUKNOUIPZUJQIJZUKULCQIULCDARTQEDARSUAUBUCUMQUKUDUKUEUFUG
+      UH $.
+
+    $( The base set of a restriction to ` A ` is a subset of ` A ` .
+       (Contributed by SN, 10-Jan-2025.) $)
+    ressbasss2 $p |- ( Base ` R ) C_ A $=
+      ( cbs cfv cin ressbasssg inss1 sstri ) CGHABIAABCDEFJABKL $.
```

Comment:
This theorem appears to be identical with `ressbasss`.

Reply:
ressbasss corresponds to the result `C_ B`, instead of `C_ A`

Reply:
@icecream17 you are right, I moved my "thumps up" emoji to your remark :-).